package.json

```diff
@@ -6,7 +6,7 @@
   "type": "module",
   "scripts": {
     "predeploy": "npm run build",
-    "deploy": "gh-pages -d build",
+    "deploy": "gh-pages -d dist",
```

Comment:
Should have noticed this on the call, but this should be the same name as the directory that gets created when you run `nom run build`, which in our case is `dist/`